bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/ghost/AsterismTypeConverters.scala

```diff
@@ -29,5 +29,8 @@ object AsterismTypeConverters {
       (GhostHigh, GhostHighResolutionTargetPlusSky, GhostStandard) -> GhostTargetPlusSky,
       (GhostHigh, GhostHighResolutionTargetPlusSky, GhostPRV) -> GhostHighResolutionTarget,
 
-      (GhostPRV, GhostHighResolutionTarget, GhostStandard) -> GhostSingleTarget)
+      (GhostPRV, GhostHighResolutionTarget, GhostStandard) -> GhostSingleTarget,
+
+      // This oddly happens sometimes?
+      (GhostStandard, GhostHighResolutionTarget, GhostHigh) -> GhostHighResolutionTarget)
```

Comment:
What is odd, I don't understand what is happening here.

Reply:
* Okay, `println`s removed. (I need to be more diligent in checking that.)
* Name of method changed as per your recommendation. 👍 
* The odd: We should never be in a state where we are in standard resolution with asterism type `GhostHighResolutionTarget`, but for some reason it has come up twice. I haven't been able to figure it out or reproduce it reliably. Maybe I need to be doing something on the Swing thread?

Reply:
(And by twice, I mean twice in hundreds of mode switches. I should probably leave it as an error since it should never happen, and then maybe we can track it down.)